snooty/postprocess.py

```diff
@@ -2194,7 +2194,7 @@ def generate_metadata(cls, context: Context) -> n.SerializedNode:
             k: (
                 context[TocTitleHandler].get_title(f"/{k}")
                 if f"/{k}" in context[TocTitleHandler].slug_title_mapping
-                else v[0].get_text()
+                else "".join(node.get_text() for node in v)
```

Comment:
should this have a space to separate text from multiple inline nodes

Reply:
Hmm, what do you mean? If a node has multiple nodes, then there's always a space somewhere in either a text value or the inline node's value.